src/glyphs.js

```diff
@@ -140,4 +176,53 @@ router.post('/:_id/gauges/:index/cache', (req, res) => {
   }
 });
 
+//Used to updated a message attached to a gauge
+router.post('/:_id/gauges/:index/messages/:num', (req, res) => {
+  const processed = processMessageRequest(req);
+
+  if (processed.errors.length > 0) {
+    res.json({
+      'errors': processed.errors
+    });
+  } else {
+    db.collection.updateOne(
+      {
+        _id: req.params._id,
+        [`view.gauges.${req.params.index - 1}.messages.${req.params.num - 1}`]: { $exists: true }
```

Comment:
This is just string interpolation, so let's bring `view.gauges.${req.params.index - 1}.messages` out into its own variable.  It will make it more concise.